src/test/java/steps/NavigationStep.java

```diff
@@ -0,0 +1,33 @@
+package steps;
+
+import org.openqa.selenium.WebElement;
+import pages.MainPage;
+import pages.NewsPage;
+
+public class NavigationStep {
+
+    MainPage mainPage = new MainPage();
```

Comment:
Don't forget about modifiers, which one should be used here and why?

Reply:
fixed